free-courses-vi.md

```diff
@@ -189,6 +189,7 @@
 
 * [SQL Cơ Bản](https://www.codehub.com.vn/SQL-Co-Ban)
 * [SQL Cơ Bản](https://www.youtube.com/watch?v=2fanjSYVElY&list=PL33lvabfss1xnFpWQF6YH11kMTS1HmLsw) - Kteam
+* [SQL cơ bản và nâng cao](https://www.youtube.com/watch?v=AGVNe9FPH04&list=PLE1qPKuGSJaDkQQB5vK7t7-PRIVjtqeHB) - thân triệu
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

```suggestion
* [Học SQL Cơ Bản](https://www.youtube.com/playlist?list=PLE1qPKuGSJaDkQQB5vK7t7-PRIVjtqeHB) - thân triệu
```